libs/components/lists/testing/src/modules/filter/filter-inline-harness.ts

```diff
@@ -0,0 +1,57 @@
+import { HarnessPredicate } from '@angular/cdk/testing';
+import { SkyComponentHarness } from '@skyux/core/testing';
+
+import { SkyFilterInlineHarnessFilters } from './filter-inline-harness-filters';
+import { SkyFilterInlineItemHarness } from './filter-inline-item-harness';
+import { SkyFilterInlineItemHarnessFilters } from './filter-inline-item-harness-filters';
+
+/**
+ * Harness to interact with a filter inline component in tests.
```

Comment:
can we change `component` to `wrapper` here to match docs

Reply:
these jsdocs are okay because its in line with what other components (see `toolbar-section-harness.ts`). Unlike `toaster` harness which is not documented so needed more documentation